vendor/revisions.json

```diff
@@ -1,5 +1,5 @@
 {
-    "postgres-v16": "389ce36b4b3da7aa654a25e1b3f10b641319a87f",
-    "postgres-v15": "74cfe3e681836747a31fdbd47bdd14b3d81b0772",
-    "postgres-v14": "5d5cfee12783f0989a9c9fe13bb40b5585812568"
+    "postgres-v16": "e6c1ece2efbf52277565936925ae43599ce51730",
+    "postgres-v15": "327f4e5e7db1050ce1b79e19f3d27ef1cff7acca",
+    "postgres-v14": "80932ee1ae0b8e645390546ed104c357cddc7d0c"
```

Comment:
Just because I think it is convenient and useful to report LFC access stats in `explain analyze` output.
Yes, `bufread - page server read ~= LFC hits`, `page server read ~= LFC misses`. But the problem is that user does not know number of PS reads!  We can see such metric in Grafana, but not for particular query. 
This is why I want to make this information available to user in most convenient way (by means of explain).